### DISCUSSION TITLE
Unable to parse when changing anything in design tab.

### DISCUSSION THREAD

**Discussion by jaroldsabillon** [eclipse/windowbuilder#351] [Q&A] [answered]
Any small change even resizing the window gives me an error.

**Comment by wimjongman** [comment, marked as answer]
Fixed by upgrading Eclipse to 22-09

<strike>You must give much much more information.</strike>

**Reply by jaroldsabillon** [reply]
I had just installed windowbuilder plugin and created a new project with it. I was using Eclipse 22-06 and tried multiple java versions and all I was getting "Unable to parse source code" whenever making any kind of changes in the design tab. I fixed this by updating to Eclipse 22-09. Just in case anyone is having the same issue.

**Comment by CarlosRepositories** [comment]
Hello everyone

When I try to do anything in my project with windows builder, the Eclipse IDE shows the error message: "Unable to parse source. WindowBuilder was not able to parse source setTitle("Locadora de Veiculos"). Why did that happen? Could you help me?
![image](https://user-images.githubusercontent.com/70611136/201503459-ab4113f5-4749-4c19-a030-9129dfbc8132.png)
